src/components/NavigationChecker/NavigationChecker.js

```diff
@@ -9,8 +10,8 @@ type Config = {}
 type State = { isActive: boolean }
 
 export const NavigationChecker = (
-  WrappedComponent: React.AbstractComponent<Config>
-): React.AbstractComponent<Config> => class extends React.Component<Config, State> {
+  WrappedComponent: AbstractComponent<Config>
```

Comment:
Why not `ComponentType`?

Reply:
it is a react type

Reply:
I have imported it from the react package

Reply:
`ComponentType` is react type too